src/NServiceBus.Core/Routing/RoutingSettings.cs

```diff
@@ -13,22 +14,28 @@ public class RoutingSettings : ExposeSettings
         internal RoutingSettings(SettingsHolder settings)
             : base(settings)
         {
-            Mapping = new RoutingMappingSettings(settings);
         }
 
         /// <summary>
-        /// Allows customizing advanced routing settings.
-        /// </summary>
-        public RoutingMappingSettings Mapping { get; }
-
-        /// <summary>
         /// Adds a static unicast route.
         /// </summary>
         /// <param name="messageType">Message type.</param>
-        /// <param name="destination">Destination endpoint.</param>
-        public void RouteToEndpoint(Type messageType, string destination)
+        /// <param name="destinationEndpoint">Destination endpoint.</param>
+        public void RouteTo(Type messageType, string destinationEndpoint)
```

Comment:
I'm pretty sure that someone using msmq would easily enter "endpointname@machine". Not saying this can't happen with `RouteToEndpoint` but it's a bit more clear on the expected parameter.